src/objects/InstancedMesh.js

```diff
@@ -258,7 +259,14 @@ class InstancedMesh extends Mesh {
 
 	dispose() {
 
-		this.dispatchEvent( { type: 'dispose' } );
```

Comment:
This line should not be removed. In `WebGLObjects` there is a listener that relies on this event.

Reply:
Ah I see! Done 👍